### DISCUSSION TITLE
评论过长

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#448] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/113572850/195502696-ab74bde7-8243-44ee-9d20-9565aaf69440.png)
@banned-historical-archives  这怎么办？

**Comment by banned-historical-archives** [comment, marked as answer]
那只能手动加到代码里了

**Reply by PirateBook1** [reply]
有可能拆成多个楼吗

**Reply by banned-historical-archives** [reply]
不能，或者你分批校对
